src/main/java/ladder/controller/LadderController.java

```diff
@@ -9,9 +9,31 @@
 public class LadderController {
 
 	public static void start() {
-		Ladder ladder = new Ladder(new PointConnectRandomly(), new Participants(InputView.receivePeopleName()),
-				InputView.receiveLadderHeight());
+		Ladder ladder = createAndPrintLadder();
+		showResult(ladder);
+	}
+
+	private static Ladder createAndPrintLadder() {
+		LadderInput ladderInput = new LadderInput(InputView.receivePeopleName(),
+				InputView.receiveLadderResult());
+		Ladder ladder = new  Ladder(new PointConnectRandomly(), ladderInput, InputView.receiveLadderHeight());
 		ResultView.printLadder(ladder);
+		return ladder;
+	}
+
+	// TODO: 2019-11-13 여기를 리팩토링 할 수 있을까?
+	//   요구사항 : ALL 이 입력되면 전체 결과를 출력하고 그만하기, 다른 이름이 입력되면 결과를 출력하고 계속 반복하기
+	//   반복하다가 조건부로 멈춰야하기 때문에 무한루프 + if + break 를 쓸 수 밖에 없지 않을까
+	private static void showResult(Ladder ladder) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
	private static void showResult(LadderExecutionResult results) {
```

Reply:
오 훨씬 깔끔해지네요!! 감사합니다 :)